tests/config/browserTest.ts

```diff
@@ -57,6 +58,18 @@ type BrowserTestTestFixtures = PageTestFixtures & {
 };
 
 const test = baseTest.extend<BrowserTestTestFixtures, BrowserTestWorkerFixtures>({
+  loopback: [async ({ channel }, use) => {
+    if (channel === 'webkit-wsl' && execSync('wsl -d playwright --cd /home/pwuser wslinfo --networking-mode').includes('nat'))
+      return await use(execSync('wsl -d playwright --cd /home/pwuser ip route show', { encoding: 'utf8' }).trim().split('\n').find(line => line.includes('default'))?.split(' ')[2]);
+    return await use(undefined);
+  }, { scope: 'worker', timeout: 10000 }],
+
+  loopback2: [async ({ channel, loopback }, use) => {
+    if (channel === 'webkit-wsl' && loopback)
+      return await use(`${loopback}.nip.io`);
```

Comment:
Can we avoid using `nip.io`?

Reply:
I don't think so, we use it today [already](https://github.com/search?q=repo%3Amicrosoft%2Fplaywright%20%2Fnip.io%2F&type=code).